lib/views/alaveteli_pro/batch_request_authority_searches/_beginner_batch_pop_up.html.erb

```diff
@@ -0,0 +1,62 @@
+  <div class="modal-container">
+    <div class="js-carousel" role="region" aria-label="Featured content" aria-roledescription="carousel" carousel-current-slide="0">
+      <div class="js-carousel__wrapper">
+        <div class="js-carousel__item" js-carousel-order="0" aria-roledescription="slide" aria-label="Slide 1 of 2">
+          <div class="modal-header">
+            <h2 class="modal-title">Responsible use of Batch</h2>
+          </div>
+          <p>
+            <em>Remember that multiple people are going to spend time answering your request. Targeting lots of authorities can swamp both them and you with unnecessary work and information you can’t use.</em>
+          </p>
+          <p>
+            <span><strong>Write a targeted request:</strong> A clear, specific, and proportionate request is more likely to get you usable information quickly. Vague or excessive demands risk refusal, delay, or inconsistent results that are difficult to compare.</span>
+          </p>
+          <p>
+            <span><strong>Send a pilot:</strong> Trial the request on a small sample of authorities and wait for responses before sending to the full list. This helps you check that they’re likely to hold the information you’re asking for and spot unclear wording, overly broad scope, or practical snags early – saving you and the authorities from wasted effort.</span>
+          </p>
+          <p>
+            <span><strong>Choose recipients carefully:</strong> Before making a request to a large number of authorities, try to find out if the information is compiled centrally, or pick a representative sample so that the project stays manageable and the results are still meaningful. Make sure the request is relevant to all authorities that you select.</span>
+          </p>
+        </div>
+
+        <div class="js-carousel__item" js-carousel-order="1" aria-roledescription="slide" aria-label="Slide 2 of 2">
+          <div class="modal-header">
+            <h2 class="modal-title">Just a reminder</h2>
+          </div>
+            <p>
+              <em>Pro users are still bound by our <%= link_to 'House Rules', help_house_rules_path %>, which includes making <%= link_to 'responsible requests', help_requesting_path(anchor: 'responsible') %>.</em>
+            </p>
+            <div class="read-container js-read-container">
+              <p class="read-container__announcement"><strong>Please read WhatDoTheyKnow's rules before continuing:</strong></p>
+              <%= render partial: 'help/house_rules/list_of_house_rules' %>
+            </div>
+        </div>
+      </div>
+
+      <div class="carousel-nav">
+        <button class="button js-carousel-button-previous" aria-label="Previous slide">
+          ← Previous
+        </button>
+        <button class="button js-carousel-button-next" aria-label="Next slide">
+          Next →
+        </button>
+        <button class="modal-close" aria-label="close modal and continue">Continue</button>
```

Comment:
This style looks broken (font family, font weight, border radius):

<img width="344" height="77" alt="Screenshot 2025-09-16 at 11 31 45" src="https://github.com/user-attachments/assets/0a158652-a843-428a-a2f0-a86b07efbddb" />